core/res/res/values/aosip_config.xml

```diff
@@ -116,4 +116,7 @@
     <!-- Whether device supports internal audio recording -->
     <bool name="config_hasInternalAudioRecordingSupport">true</bool>
 
+    <!-- Whether to use scaled brightness (set -1 to disable) -->
+    <integer name="config_useScaleBrightness">-1</integer>
```

Comment:
Do you have an overlay for that in your DT?

Reply:
Yes : https://github.com/nullxception/device_xiaomi_raphael/commit/db1edae0e031c5ab55a13cfc540aa1c9604a2e7e